### ISSUE TITLE
Time slip - ship moves abruptly on use of time warp after being teleported to orbit.

### ISSUE THREAD

**Issue by Ezriilc** [Ezriilc/HyperEdit#14]
To reproduce:
Edit orbit to Kerbin @ 100k, then to Mun at same.
Start/stop time warp to see craft jump position along orbit.

**Comment by Ezriilc** [comment]
I believe this issue can be worked around by a save/load.

**Comment by Ezriilc** [comment]
I confirmed this still exists in v 1.5.6.

**Comment by fronbow** [comment]
I'm having trouble recreating this.
Are you in map view for all of this or vessel view?

**Comment by Ezriilc** [comment]
Oh yes, sorry.  It's in map view.
I suspect this has something to do with the status of the vessel after HE edits an orbit, similar to "on rails" or off, perhaps.  If I'm not mistaken, changing scene and back eliminates it.

**Comment by fronbow** [comment]
Yes, if, like me, you do stuff in vessel view (I prefer to see crashes than hear them ;) ) then this doesn't seem to be a reproducible problem.
It could well be a KSP physics thing related to mapview i.e. out of our hands/scope.

**Comment by chambm** [comment]
Still exists for me too. It really screws up MechJeb maneuvers when auto-warp is on. IIRC it happened for me both in map view and vessel view. I didn't try reloading because I wanted to keep my revert option but I'll report back later if that changes it.
